### PR TITLE
feat: delete uploaded licence doc in driver

### PR DESCRIPTION
**Pull request by PratikGanjareFleetHawks**
## Change Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist: (Check only applicable)

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/app/pages/fleet/drivers/driver-detail/driver-detail.component.html [hunk 1]
```diff
@@ -1201,12 +1201,12 @@ <h5 class="modal-title">Driver Details</h5>
                       >
                     </div>
                     <div class="col-lg-3 text-right">
-                      <!--<a-->
-                      <!--  href="javascript:;"-->
```
Comment (Daljit-fleetHawks) [inline src/app/pages/fleet/drivers/driver-detail/driver-detail.component.html:1205]: @PratikGanjareFleetHawks Please remove commented code

File: src/app/pages/fleet/drivers/add-driver/add-driver.component.html [hunk 2]
```diff
@@ -1048,8 +1044,8 @@ <h4 class="text-4 mt-0 mb-0 text-dark">{{ pageTitle }}</h4>
                                 (click)="setPDFSrc(doc.path)">Document {{ d + 1 }}</a>
                             </div>
                             <div class="col-lg-3 text-right">
-                              <!--<a href="javascript:;" class="btn btn-deafault pull-right btn-xs"-->
```
Comment (Daljit-fleetHawks) [inline src/app/pages/fleet/drivers/add-driver/add-driver.component.html:1051]: @PratikGanjareFleetHawks Please remove commented code. Its repeating

**Review by kunalfleethawks — APPROVED** [review]

File: src/app/pages/fleet/drivers/driver-detail/driver-detail.component.html (hunk 1 above)
Comment (PratikGanjareFleetHawks) [reply src/app/pages/fleet/drivers/driver-detail/driver-detail.component.html:1205]: @Daljit-fleetHawks commented code is not there.

File: src/app/pages/fleet/drivers/add-driver/add-driver.component.html (hunk 2 above)
Comment (PratikGanjareFleetHawks) [reply src/app/pages/fleet/drivers/add-driver/add-driver.component.html:1051]: @Daljit-fleetHawks commented code is not there .

**Pull request merged**